model/keyvalue.go

```diff
@@ -144,6 +144,27 @@ func (kv *KeyValue) AsString() string {
 	}
 }
 
+// AsString returns a string representation of the value.
+func (kv *KeyValue) AsString() string {
```

Comment:
the two functions are nearly identical, can we have them both call into a private helper with a `truncate bool` parameter?